client/src/App/Header/Header.tsx

```diff
@@ -45,6 +51,8 @@ const Header: React.FC = () => {
                         <MenuDesktop
                             loadingState={loadingState}
                             items={menuItems}
+                            currentUser={currentUser}
+                            onLogoutCb={onLogoutCb}
```

Comment:
Don't use abbreviations like `Cb`. If you want to have full name please change it to `callback`, but in my opinion `onLogout` as a name is sufficient.